cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2403,23 +2403,23 @@
 
   <property>
     <name>system.runtime.monitor.retry.policy.max.retries</name>
-    <value>1000</value>
+    <value>2147483647</value>
     <description>
       The maximum number of retries to attempt before aborting
     </description>
   </property>
 
   <property>
     <name>system.runtime.monitor.retry.policy.max.time.secs</name>
-    <value>120</value>
+    <value>2147483647</value>
```

Comment:
Does this mean it can get stuck on a bad message, and fail to process messages after it, if it keeps retrying on that bad message?

Reply:
The runtime monitor itself shouldn't give-up, otherwise it'll just stop monitoring. The message processor should handle bad messages and skip them.